tests-manual/debugger/README.md

```diff
@@ -0,0 +1,45 @@
+Debugger Command Line Test Programs
+===================================
+
+This directory contains a set of utilities used to manually test the pipeline debugger features. These tests should be run against a live system with the debugger facilities built into it.
+
+To start testing;
+
+  - Change directory to this directory
+
+  - Type the command
+
+    .. code-block:: console
```

Comment:
None of the _code-blocks_ in this file are rendering correctly. I am not sure why. I did notice there were tabs in front of "_.. code-block._" I think RST requires 4 spaces.